### PR TITLE
Fix display of subform repeatable on IE

### PR DESCRIPTION
**Pull request by ventsonge**
Pull Request for Issue #25194 .

### Summary of Changes
Remove the append of template at the end of the form since it's unnecessary.
add the display none to the template tag, cause it's not properly recognize on IE

### Testing Instructions
Create a subform and try to display it on IE (tested with 11.0)

### CODE REVIEW

File: layouts/joomla/form/field/subform/repeatable-table.php [hunk 1]
```diff
@@ -118,21 +118,23 @@ class="btn btn-mini button btn-success group-add group-add-<?php echo $unique_su
...
-							'form' => $tmpl,
-							'basegroup' => $fieldname,
-							'group' => $fieldname . 'X',
-							'buttons' => $buttons,
-							'unique_subform_id' => $unique_subform_id,
-						)
-					)
-				); ?></template>
-			<?php endif; ?>
+            <?php if ($multiple) : ?>
```
Comment (Quy) [inline layouts/joomla/form/field/subform/repeatable-table.php]: Please use tabs and not spaces to indent.

**Comment by Quy** [comment]
Pinging @Fedik

**Comment by Fedik** [comment]
just make it display:none, inline, that should be enough, and without overhead

```html
<template class="subform-repeatable-template-section" style="display:none">
...
</template>
```

**Comment by Quy** [comment]
@Fedik Your suggestion results in:

>PHP Warning:  Invalid argument supplied for foreach() in \libraries\src\Form\Form.php on line 1558

**Comment by Quy** [comment]
Please test PR #25321<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/25200">issues.joomla.org/tracker/joomla-cms/25200</a>.</sub>

**Pull request closed without merging**